components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/behavior/BrowserToolbarBottomBehavior.kt

```diff
@@ -92,20 +95,11 @@ class BrowserToolbarBottomBehavior(
         axes: Int,
         type: Int
     ): Boolean {
-        return if (shouldScroll && axes == ViewCompat.SCROLL_AXIS_VERTICAL) {
-            startedScroll = true
-            shouldSnapAfterScroll = type == ViewCompat.TYPE_TOUCH
-            snapAnimator.cancel()
-            true
-        } else if (engineView?.getInputResult() == EngineView.InputResult.INPUT_RESULT_UNHANDLED) {
-            // Force expand the toolbar if event is unhandled, otherwise user could get stuck in a
-            // state where they cannot show the toolbar
-            snapAnimator.cancel()
-            forceExpand(child)
-            false
-        } else {
-            false
+        browserToolbar?.let {
+            return startNestedScroll(axes, type, child)
```

Comment:
nit (here and below): If we only do a null check and do not use the value here, then I think an `if` is just easier to read and understand and using `let` is just over-complicated? :)